composer.json

```diff
@@ -95,13 +94,10 @@
         "pulse00/ffmpeg-bundle": "^0.6",
         "symfony/browser-kit": "^3.4",
         "symfony/css-selector": "^3.4",
-        "symfony/debug": "^3.4",
```

Comment:
This is used in the [front controllers](https://github.com/sulu/sulu/blob/develop/web/app_dev.php#L17).

Reply:
fixed